src/App.vue

```diff
@@ -1,15 +1,8 @@
 <template lang='pug'>
 	v-app
-		v-app-bar(app, color='green', dark, dense)
-			v-toolbar-title JavaScript Code Smells Detector
-			v-spacer
-			v-btn(to='/upload', text) Code upload
-			v-btn(to='/editor', text) Code editor
-			v-btn(to='/', text) Home page
-
+		base-app-bar

```

Comment:
It looks like using separate components breaks some styles in our application (i've noticed that footer is different).